src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -55,6 +61,7 @@ const Page = ({
   chain: chainFromProps,
   address: addressFromProps,
 }: Omit<Props, "fallback">) => {
+  const { isOpen, onClose, onOpen } = useDisclosure() // for EditRewardDescriptionModal
```

Comment:
Let's remove the comment and rename the variables instead, so it's easier to find them in the code, like this:
```ts
const {
  isOpen: isEditRewardDescriptionModalOpen,
  ...
} = useDisclosure()
```

Reply:
Yes I thought about that too